server.js

```diff
@@ -33,11 +33,7 @@ mongoose
     useFindAndModify: false,
     useCreateIndex: true
   })
-  .then(() => {
-    console.log('MongoDB is connected')
-  })
-
```

Comment:
why are you deleting this?

Reply:
It didn't seem necessary and was messing up the output of my tests. If you don't get an error you've connected successfully, right?

Reply:
it helps us know we connected successfully. i suggest we keep it

Reply:
Ben seems to be suggesting we remove it. Are you happy to, or still want to discuss further?

Reply:
i'm good now. we can take it out